domain-module/review/src/main/java/com/romanticpipe/reviewcanvas/service/ProductValidator.java

```diff
@@ -0,0 +1,29 @@
+package com.romanticpipe.reviewcanvas.service;
+
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
+import com.romanticpipe.reviewcanvas.domain.Product;
+import com.romanticpipe.reviewcanvas.domain.Review;
+import com.romanticpipe.reviewcanvas.exception.BusinessException;
+import com.romanticpipe.reviewcanvas.exception.ProductErrorCode;
+import com.romanticpipe.reviewcanvas.repository.ProductRepository;
+import com.romanticpipe.reviewcanvas.repository.ReviewRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class ProductValidator {
+
+	private final ProductRepository productRepository;
+
+	public Product validByProductId(String productId){
+		Optional<Product> productOptional = productRepository.findById(productId);
+		if (productOptional.isEmpty())
+			throw new BusinessException(ProductErrorCode.PRODUCT_NOT_FOUND);
+		return productOptional.get();
```

Comment:
```suggestion
		return productRepository.findById(productId)
		            .orElseThrow(() -> new BusinessException(ProductErrorCode.PRODUCT_NOT_FOUND));
```
p3: optional에서 제공하는 함수와 람다를 활용하여 간단하게 바꿀 수 있습니다! 혹시 이렇게 바꾸어보는 것은 어떠신가요?

Reply:
모두 수정하였습니다.